strapi/strapi
# fix(content-releases): dont allow update on a released release

### What does it do?

Remove the options menu from release details view when release is released. Also add validations for this cases on backend.

### How to test it?

1. Go to a release and publish it
2. Now you should not see the options menu
3. If you try to make changes to the release directly by the API, it should throw an error if it's released
